### ISSUE TITLE
AttributeError: 'tuple' object has no attribute 'request'

### ISSUE THREAD

**Issue by amabdallah** [hydroshare/hs_restclient#123]
Hi @Castronova and @sblack-usu 

When I run this very basic connection, I get the error below. My code used to work fine last year but not anymore. Any suggestions?

```
username = 'myusername'
password = 'mypass'

from hs_restclient import HydroShare, HydroShareAuthBasic
auth = HydroShareAuthBasic(username=username, password=password)
hs = HydroShare(auth=auth)
for resource in hs.resources():
    print(resource)
```

>   File "C:\Python38\lib\site-packages\hs_restclient\generators.py", line 12, in resultsListGenerator
>     raise HydroShareHTTPException((url, 'GET', r.status_code, params))
>   File "C:\Python38\lib\site-packages\hs_restclient\exceptions.py", line 61, in __init__
>     self.url = response.request.url
> AttributeError: 'tuple' object has no attribute 'request'

**Comment by amabdallah** [comment]
@Castronova

**Comment by Castronova** [comment]
@amabdallah Which version of the `hs_restclient` are you using? Can you run the following command:

```
python -c "import hs_restclient; print(hs_restclient.__version__)"
```

**Comment by Castronova** [comment]
@sblack-usu  @amabdallah  I was able to reproduce this error in version 1.3.6. 

```
usr = 'username'
psk = 'password'

from hs_restclient import HydroShare, HydroShareAuthBasic
auth = HydroShareAuthBasic(username=usr, password=psk)
hs = HydroShare(auth=auth)

resources = hs.resources()

# try to print the first resource.
next(resources)

```


```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/castro/miniconda2/envs/hsrest/lib/python3.6/site-packages/hs_restclient/generators.py", line 12, in resultsListGenerator
    raise HydroShareHTTPException((url, 'GET', r.status_code, params))
  File "/Users/castro/miniconda2/envs/hsrest/lib/python3.6/site-packages/hs_restclient/exceptions.py", line 61, in __init__
    self.url = response.request.url
AttributeError: 'tuple' object has no attribute 'request'
```

**Comment by sblack-usu** [comment]
This has already been fixed in develop, it just needs a release.  I can do the release today.

**Comment by amabdallah** [comment]
Thanks, @Castronova and @sblack-usu . Sorr I should've listed the version. It was the latest one as posted here
https://hs-restclient.readthedocs.io/en/latest/

**Comment by sblack-usu** [comment]
Just released the 1.3.7 which will provide a proper exception
![image](https://user-images.githubusercontent.com/35820390/82258153-8cdf5c00-9916-11ea-8076-f7965f5d2efd.png)

**Comment by amabdallah** [comment]
@sblack-usu would you please propagate this release here? https://hs-restclient.readthedocs.io/en/latest/

I'm having issues installing the latest release  v1.3.7 from GitHub. It is still pulling the 1.3.6
